<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cuTPS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Testpage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Server IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtServerIP">
            <property name="placeholderText">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runTestsButton">
            <property name="text">
             <string>Run All Tests</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRunTest3">
            <property name="text">
             <string>Add Invoice Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRunTest2">
            <property name="text">
             <string>Get Content Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRunTest1">
            <property name="text">
             <string>Add Content Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetIP">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="resultsListWidget"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LoginPage">
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>199</width>
          <height>99</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PasswordBox"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="BtnClear">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnLogin">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="UsernameBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="MainStudent">
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>161</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>391</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>301</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Congratulations you logged in</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnLogout">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>10</y>
          <width>88</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
